.pre-commit-config.yaml

```
@@ -37,25 +37,13 @@ repos:
         exclude_types: [json, pofile]
         exclude: 'changelog/|py.typed|disnake/bin/COPYING|.github/PULL_REQUEST_TEMPLATE.md|.github/CODEOWNERS|LICENSE|MANIFEST.in'
 
-  - repo: https://github.com/pycqa/isort
-    rev: 5.12.0
-    hooks:
-      - id: isort
-        args: ["--profile", "black", "--extend-skip", "examples"]
-        name: "run isort in all files"
-        exclude: ^examples/
-      - id: isort
-        args: ["--profile", "black", "--thirdparty", "disnake"]
-        name: "run isort in examples"
-        files: ^examples/
-
   - repo: https://github.com/psf/black-pre-commit-mirror
     rev: 23.9.1
     hooks:
       - id: black
         name: "run black in all files"
 
   - repo: https://github.com/charliermarsh/ruff-pre-commit
-    rev: v0.0.292
+    rev: v0.3.3
     hooks:
       - id: ruff
```

```suggestion
      - id: ruff
        args: [--fix, --fixable=I]
```
This autofixes the import order like the isort step did before